app/models/card.rb

```diff
@@ -14,4 +14,10 @@ def the_text_cannot_be_the_same
   def shift_date
     self.review_date = Date.today + 3.days
   end
+
+  def check(original_text)
+    original_text == self.original_text
+  end
+  
+  scope :review_cards, -> { where('review_date <=?', Date.today) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.